mmv1/templates/terraform/examples/network_security_server_tls_policy_basic.tf.erb

```diff
@@ -17,16 +16,6 @@ resource "google_network_security_server_tls_policy" "<%= ctx[:primary_resource_
         target_uri = "unix:mypath"
       }
     }
-    client_validation_ca {
-      grpc_endpoint {
-        target_uri = "unix:abc/mypath"
-      }
-    }
-    client_validation_ca {
-      certificate_provider_instance {
-        plugin_instance = "google_cloud_private_spiffe"
-      }
-    }
```

Comment:
why are these removed?

Reply:
Only one client_validation_ca is allowed, this is what GCP error said.

Reply:
Thanks! Looks like that is tracked in https://github.com/hashicorp/terraform-provider-google/issues/18103